src/dpl/README.md

```diff
@@ -9,18 +10,25 @@ Features:
 
 ## Commands
 
+### Detailed Placement
+
 The `detailed_placement` command performs detailed placement of instances
 to legal locations after global placement.
 
+```tcl
+detailed_placement
+    [-max_displacement disp|{disp_x disp_y}]
+    [-disallow_one_site_gaps]
 ```
-set_placement_padding -global|-instances insts|-masters masters
-                      [-left pad_left] [-right pad_right]
-detailed_placement [-max_displacement disp|{disp_x disp_y}]
-check_placement [-verbose]
-filler_placement [-prefix prefix] filler_masters
-remove_fillers
-optimize_mirroring
-```
+
+#### Options
+
+| Switch Name | Description | 
+| ----- | ----- |
+| `-max_displacement` | How far an instance (in microns) can be moved when finding a site where it can be placed. Either set one value for both directions or set {disp_x disp_y} for individual directions. (default 0, 0, integer). |
```

Comment:
```suggestion
| `-max_displacement` | Max distance that an instance can be moved (in microns) when finding a site where it can be placed. Either set one value for both directions or set `{disp_x disp_y}` for individual directions. (default `{0, 0}`, integer). |
```